docs/localization-translation/countries-and-subdivisions.md

```diff
@@ -0,0 +1,15 @@
+---
+post_title: Updating Countries and Subdivisions
+menu_title: Translating WooCommerce
```

Comment:
Open to thoughts on this ... I wanted to place this in the `localization-translation` section, and so am reusing the existing menu title. However, that menu title implies this is about translation.

Not sure if that's problematic or not. Happy to modify this, though.